CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 1.4.1
+
+## Deprecations
```

Comment:
I just read `CHANGELOG.md` of Realm Java <https://github.com/realm/realm-java/blob/master/CHANGELOG.md> It looks to me like`### Deprecated` is more suited.